executor/load_data.go

```diff
@@ -682,6 +811,7 @@ func (e *LoadDataWorker) processStream(
 		}
 
 	TrySendTask:
+		pos, _ := seeker.Seek(0, io.SeekCurrent)
```

Comment:
It's better to check err.

Reply:
It will print a lot of error in log, and the final effect should be equal to only use `pos`

Reply:
Is this operation easy to fail?

Reply:
if it fail once for some unknown reason, it will always fail for this file. I can add a flag to log the error in next commit

Reply:
So can we fail fast when error?

Reply:
progress is not as important as loading the real data, I don't want it fail